### PR TITLE
Regression: Fix issues with ContactHistoryItem

### PR DESCRIPTION
**Pull request by MartinSchoeler**
This fixes:

- [x] Empty closed message issue
- [x] Missing Dates on contact history item

I've also changed the layout a bit, to better mirror the future design for this part of the UI

Screen Shot 2022-11-29 at 2 24 28 PM

OC-448
OC-449

### CODE REVIEW

File: apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx [hunk 1]
```diff
@@ -21,8 +22,9 @@ type ContactHistoryItemProps = {
 
 function ContactHistoryItem({ history, setChatId, ...props }: ContactHistoryItemProps): ReactElement {
 	const t = useTranslation();
-	const formatDate = useTimeAgo();
+	const formatDate = useFormatDateAndTime();
 	const username = history.servedBy?.username;
+	const hasClosingMessage = history.closingMessage && history.closingMessage.msg !== '';
```
Comment (tiagoevanp) [inline apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]:
```suggestion
	const hasClosingMessage = history.closingMessage?.msg;
```

Comment (tiagoevanp) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: Why not this?

**Review by tiagoevanp — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx (hunk 1 above)
Comment (MartinSchoeler) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: There is a possibility of the room being closed with the closing message as `''` the closing message object will only not exist when a room is closed and the whole comment feature is turned off on the omni configurations

Comment (ggazzo) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: how about `str.trim()` ?

Comment (tiagoevanp) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: BTW: I forgot `!!` on the beginning... casting to boolean...

Comment (tiagoevanp) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: image

Comment (ggazzo) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx]: how about str.trim() ?

File: apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx [hunk 2]
```diff
@@ -21,8 +22,9 @@ type ContactHistoryItemProps = {
 
 function ContactHistoryItem({ history, setChatId, ...props }: ContactHistoryItemProps): ReactElement {
 	const t = useTranslation();
-	const formatDate = useTimeAgo();
+	const formatDate = useFormatDateAndTime();
 	const username = history.servedBy?.username;
+	const hasClosingMessage = !!history.closingMessage?.msg;
```
Comment (ggazzo) [inline apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: how about str.trim() ?

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx (hunk 2 above)
Comment (tiagoevanp) [reply apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx:27]: `     ` in case of space messages, right?

**Review by tiagoevanp — APPROVED** [review]

**Pull request merged**